build/build.xml

```diff
@@ -24,6 +24,21 @@
     <echo message="The tools.jar file is not required to build Processing." />
   </target>
 
+  <!-- detetect Raspberry Pi and friends -->
+  <condition property="linux-arm32" value="linux-arm32">
+    <and>
+      <equals arg1="${platform}" arg2="linux" />
+      <equals arg1="${os.arch}" arg2="arm" />
+      <equals arg1="${sun.arch.data.model}" arg2="32" />
+    </and>
+  </condition>
+
+  <target name="check-linux-arm32" if="linux-arm32">
```

Comment:
I think you still need an `antcall` or another target that depends on this one so that it's called properly. (Or am I missing that elsewhere in this PR?)